src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -0,0 +1,406 @@
+   */
+  public SubscriptionSchedule release(RequestOptions options) throws StripeException {
+    return release((Map<String, Object>) null, options);
+  }
+
+  /**
+   * Releases the subscription schedule immediately, which will stop scheduling of its phases, but
+   * leave any existing subscription in place. A schedule can only be released if its status is
+   * <code>not_started</code> or <code>active</code>. If the subscription schedule is currently
+   * associated with a subscription, releasing it will remove its <code>subscription</code> property
+   * and set the subscription’s ID to the <code>released_subscription</code> property.
+   */
+  public SubscriptionSchedule release(Map<String, Object> params) throws StripeException {
+    return release(params, (RequestOptions) null);
+  }
+
+  /**
+   * Releases the subscription schedule immediately, which will stop scheduling of its phases, but
+   * leave any existing subscription in place. A schedule can only be released if its status is
+   * <code>not_started</code> or <code>active</code>. If the subscription schedule is currently
+   * associated with a subscription, releasing it will remove its <code>subscription</code> property
+   * and set the subscription’s ID to the <code>released_subscription</code> property.
+   */
+  public SubscriptionSchedule release(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url =
+        String.format(
+            "%s%s",
+            Stripe.getApiBase(),
+            String.format("/v1/subscription_schedules/%s/release", this.getId()));
+    return request(
+        ApiResource.RequestMethod.POST, url, params, SubscriptionSchedule.class, options);
+  }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class CurrentPhase extends StripeObject {
+    @SerializedName("end_date")
+    Long endDate;
+
+    @SerializedName("start_date")
+    Long startDate;
+  }
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class Phase extends StripeObject {
+    /**
+     * A non-negative decimal between 0 and 100, with at most two decimal places. This represents
+     * the percentage of the subscription invoice subtotal that will be transferred to the
+     * application owner's Stripe account during this phase of the schedule.
+     */
+    @SerializedName("application_fee_percent")
+    BigDecimal applicationFeePercent;
+
+    /** ID of the coupon to use during this phase of the subscription schedule. */
+    @SerializedName("coupon")
+    @Getter(lombok.AccessLevel.NONE)
+    @Setter(lombok.AccessLevel.NONE)
+    ExpandableField<Coupon> coupon;
+
+    /** The end of this phase of the subscription schedule. */
+    @SerializedName("end_date")
+    Long endDate;
+
+    /** Plans to subscribe during this phase of the subscription schedule. */
+    @SerializedName("plans")
+    List<SubscriptionScheduleRevision.PhaseItem> plans;
```

Comment:
Same as before, I think this should exist in this class. Also some FUD on naming of the class. I wish the property was named `items` and not `plans` and it'd make more sense, but because it's not I'm less a fan.

Reply:
also nested in `SubscriptionSchedule`